.github/workflows/publish.yml

```diff
@@ -9,21 +9,26 @@ on:
 jobs:
   publish:
```

Comment:
Add `uses: ./.github/workflows/step_tests-pip.yml` and etc. calls, calling all steps (can avoid `static-analysis`), with appropriate requires chaining.